Sources/VLCMovieViewController.m

```diff
@@ -471,31 +485,43 @@ - (void)viewDidLayoutSubviews
+            multiSelectionFrame = (CGRect){CGPointMake(0., 0.), [_multiSelectionView proposedDisplaySize]};
+            multiSelectionFrame.origin.x = controllerPanelFrame.size.width - multiSelectionFrame.size.width;
+            multiSelectionFrame.origin.y = controllerPanelFrame.origin.y - multiSelectionFrame.size.height;
+            _multiSelectionView.frame = multiSelectionFrame;
+            _multiSelectionView.showsEqualizer = YES;
+            return;
+        }
 
-    CGRect multiSelectionFrame;
-    CGRect controllerPanelFrame = _controllerPanel.frame;;
-
-    if (UI_USER_INTERFACE_IDIOM() != UIUserInterfaceIdiomPhone) {
+        // TODO: Refactor this and remove duplicate code
+        if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])
+            _multiSelectionView.showsEqualizer = YES;
+        else
+            _multiSelectionView.showsEqualizer = NO;
+            
         multiSelectionFrame = (CGRect){CGPointMake(0., 0.), [_multiSelectionView proposedDisplaySize]};
         multiSelectionFrame.origin.x = controllerPanelFrame.size.width - multiSelectionFrame.size.width;
         multiSelectionFrame.origin.y = controllerPanelFrame.origin.y - multiSelectionFrame.size.height;
         _multiSelectionView.frame = multiSelectionFrame;
-        _multiSelectionView.showsEqualizer = YES;
-        return;
-    }
-
-    if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])) {
-        _multiSelectionView.showsEqualizer = YES;
-        multiSelectionFrame = (CGRect){CGPointMake(0., 0.), [_multiSelectionView proposedDisplaySize]};
-        multiSelectionFrame.origin.x = controllerPanelFrame.size.width - multiSelectionFrame.size.width;
-        multiSelectionFrame.origin.y = controllerPanelFrame.origin.y - multiSelectionFrame.size.height;
-    } else {
-        _multiSelectionView.showsEqualizer = NO;
-        multiSelectionFrame = (CGRect){CGPointMake(0., 0.), [_multiSelectionView proposedDisplaySize]};
-        multiSelectionFrame.origin.x = controllerPanelFrame.size.width - multiSelectionFrame.size.width;
-        multiSelectionFrame.origin.y = controllerPanelFrame.origin.y - multiSelectionFrame.size.height;
-    }
-    _multiSelectionView.frame = multiSelectionFrame;
+            
+    #else
+        CGRect videoOptionsBarFrame = CGRectMake(0., 0., 24., 120.);
+        CGFloat x,y;
+        x = controllerPanelFrame.size.width - videoOptionsBarFrame.size.width;
+        y = controllerPanelFrame.origin.y - videoOptionsBarFrame.size.height;
+        videoOptionsBarFrame = CGRectMake(x,y,videoOptionsBarFrame.size.width,videoOptionsBarFrame.size.height);
+        _videoOptionsControlBar.frame = videoOptionsBarFrame;
+        if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])) {
+            _videoOptionsControlBar.orientationAxis = UILayoutConstraintAxisHorizontal;
+        }
+    #endif
```

Comment:
Can I possibly do this in the next PR? I have made the changes and split this PR into three main PRs so it would be more convenient in the next PR to make this change while adding the proper constraints